scripts/build_python.sh

```diff
@@ -122,18 +122,20 @@ source "$CHIP_ROOT/scripts/activate.sh"
 gn --root="$CHIP_ROOT" gen "$OUTPUT_ROOT" --args="chip_detail_logging=$chip_detail_logging enable_pylib=$enable_pybindings enable_rtti=$enable_pybindings chip_project_config_include_dirs=[\"//config/python\"] $chip_mdns_arg $chip_case_retry_arg $pregen_dir_arg"
 
 # Compiles python files
+MATTER_YAMLTESTS_TARGET=$(gn ls "$OUTPUT_ROOT" --as=output //scripts:matter_yamltests_distribution._build_wheel)
```

Comment:
I believe this will as a side-effect install idl_parsing into the build environment if the flag of `install in build env` is used and that could have unintended sideffects too on future builds.

Maybe place this on a non-default flag to make if very explicit if one wants to use it.

I think `install-wheel` set to `separate` can install this, but when using `build-env` we should by default not install it (and this makes things complex ... sigh ... maybe not install by default is easiest)

Reply:
Ok I added a `--include_yamltests` flag and use that in .github/workflows/tests.yaml where the yamltests are run.